src/library/scala/util/control/NonFatal.scala

```diff
@@ -26,7 +26,7 @@ package util.control
  *   try {
  *     // dangerous stuff
  *   } catch {
- *     case NonFatal(e) => log.error(e, "Something not that bad.")
+ *     case e if NonFatal(e) => log.error(e, "Something not that bad.")
```

Comment:
this would be redundant with the next line of the comment example.

A note about which is preferred would be nice, except I hope my PR is accepted to make it a non-issue.